third_party/move/tools/move-unit-test/tests/test_sources/issue_17188.move

```diff
@@ -0,0 +1,11 @@
+module 0xc0ffee::m {
+    #[test]
```

Comment:
Could we add a few more tests like this where we error and check the message for error location?